docs/types/mapping-types.rst

```diff
@@ -221,3 +224,28 @@ the ``sum`` function iterates over to sum all the values.
             }
         }
     }
+
+.. note::
```

Comment:
I don't think this is the best place for such a comment. Please note this is the Spanish translation for the official Solidity documentation. 

If you wish to raise an issue about the clarity of the documentation it might be best to do that in the official repository https://github.com/ethereum/solidity.